drivers/ethernet/nxp_imx_netc/eth_nxp_imx_netc.c

```diff
@@ -26,6 +26,12 @@ const struct device *netc_dev_list[NETC_DRV_MAX_INST_SUPPORT];
 static int netc_eth_rx(const struct device *dev)
 {
 	struct netc_eth_data *data = dev->data;
+	struct net_if *iface_dst = data->iface;
+#if !(defined(FSL_FEATURE_NETC_HAS_SWITCH_TAG) && FSL_FEATURE_NETC_HAS_SWITCH_TAG) && \
+	defined(CONFIG_NET_DSA)
```

Comment:
I am seeing this ifdef used in many places in this driver. It is somewhat hard to read, so I would suggest making a shorter symbol that can be used instead. So something like this
```
#if !(defined(FSL_FEATURE_NETC_HAS_SWITCH_TAG) && FSL_FEATURE_NETC_HAS_SWITCH_TAG) && \
	defined(CONFIG_NET_DSA)
#define USE_DSA
#elif defined(USE_DSA)
#undef USE_DSA
#endif
```
then the checks would become much simpler and easier to read
```
#if defined(USE_DSA)
....
#endif
```

Reply:
Great. I will do that.
Thanks.